Settings.cs

```diff
@@ -11,12 +11,16 @@ public class Settings
 
         public string EditorPath { get; set; } = "";
 
+        [Obsolete]
```

Comment:
shouldnt we just delete them, no point in marking obsolete

Reply:
true